inlong-dashboard/src/ui/pages/GroupDetail/Audit/config.tsx

```diff
@@ -93,14 +93,53 @@ export const toChartData = (source, sourceDataMap) => {
 };
 
 export const toTableData = (source, sourceDataMap) => {
-  return Object.keys(sourceDataMap)
+  const map = Object.keys(sourceDataMap)
     .reverse()
     .map(logTs => ({
       ...sourceDataMap[logTs],
       logTs,
     }));
+  return getSourceDataWithPercent(source, map);
 };
 
+export const getSourceDataWithPercent = (sourceKeys, sourceMap) => {
+  const auditIds = Array.from(
+    new Set(Object.values(sourceKeys).map(({ auditId }) => parseInt(auditId))),
+  );
+  return sourceMap.map(source => {
+    for (const auditId of auditIds) {
+      if (!(auditId in source)) {
+        source[auditId] = 0;
+      }
+    }
+    let newSource = {};
+    const keys = Object.keys(source).filter(key => key !== 'logTs');
+    const firstKey = keys[0];
+    const firstValue = source[firstKey];
+    newSource[firstKey] = firstValue.toString();
+    for (let key of keys.slice(1)) {
+      if (key !== 'logTs') {
+        let diff = getDiff(firstValue, source[key]);
+        newSource[key] = `${source[key]} (${diff})`;
+      }
+    }
+    newSource['logTs'] = source['logTs'];
+    return newSource;
+  });
+};
+
+export const getDiff = (first, current) => {
+  if (first === 0) {
+    return '0%';
+  }
+  let result;
+  const diff = parseFloat(((current / first - 1) * 100).toFixed(3));
```

Comment:
The decimal point displayed on the page is of little significance. It is recommended to remove the decimal point.

Reply:
Done